src/lexer/lexer.rs

```diff
@@ -17,6 +17,10 @@ pub struct Lexer {
     /// because the first line is always at indentation level 0
     indent_stack: Vec<usize>,
     nesting: i8,
+    // This stack means we are in a fstring that started with
+    // character at the top of the stack
+    fstring_stack: Vec<String>,
+    inside_fstring_bracket: i8,
```

Comment:
Can you add a comment about `inside_fstring_bracket`? I don't understand what it is.